Meshtastic/Views/Nodes/EnvironmentMetricsLog.swift

```diff
@@ -52,12 +58,9 @@ struct EnvironmentMetricsLog: View {
 										x: .value("Time", dataPoint.time!),
 										y: .value("Temperature", dataPoint.temperature.localeTemperature())
 									)
-									.interpolationMethod(.cardinal)
+									.interpolationMethod(.catmullRom)
```

Comment:
We may want to consider trying .catmullRom. When there are large jumps in time and value in these charts, the .cardinal interpolation can create some odd curves. .catmullRom isn't perfect, but it handles some of this a bit better.